src/MergeContext.js

```diff
@@ -973,8 +973,18 @@ class PullRequest {
                 statusChecks.addOptionalStatus(new StatusCheck(st));
         }
 
+        // Returns whole check runs history for the commit.
+        // We need only the freshest checks.
         const checkRuns = await GH.getCheckRuns(this._prHeadSha());
```

Comment:
Let's move the second part of this new comment down, to the code it describes.

```suggestion
        const checkRuns = await GH.getCheckRuns(this._prHeadSha());
        // Filter out stale/older checks, assuming that check.id is greater in newer checks.
```

Reply:
Done.